specification/hdinsight/data-plane/Microsoft.HDInsight/preview/2018-11-01-preview/examples/HDI_SparkSession_Create.json

```diff
@@ -0,0 +1,37 @@
+{
+  "parameters": {
+    "endpoint": "cluster.azurehdinsight.net",
+    "X-Requested-By": "admin",
+    "sparkSessionJobRequest": {
+      "kind": "spark",
+      "proxyUser": "",
+      "jars": [],
+      "pyFiles": [],
+      "files": [],
+      "driverMemory": "4g",
+      "driverCores": 4,
+      "executorMemory": "2g",
+      "executorCores": 2,
+      "numExecutors": 2,
+      "archives": [],
+      "queue": "default",
+      "name": "jobname",
+      "conf": null,
+      "heartbeatTimeoutInSecond": 0
+    }
+  },
+  "responses": {
+    "201": {
+      "body": {
+        "id": 1,
+        "appId": "fill in here",
+        "owner": "admin",
+        "proxyUser": "",
+        "kind": "spark",
+        "state": "the state",
+        "log": [],
+        "appInfo": null
```

Comment:
Not a Swagger feedback: out of curiosity, does the server actually return `null`? Is that a significant value for you, since most of the SDK won't make a difference between "appInfo was not here" and "appInfo was here and was null"

Reply:
You are right. It will return map of key=val instead of null. I will make all examples complete.